Drv/LinuxI2CDriver/Makefile

```diff
@@ -0,0 +1,22 @@
+# This Makefile goes in each module, and allows building of an individual module library.
```

Comment:
We'll need to add a CMakeList.txt for this component too.

Reply:
Can I easily utilize a CMakeList.txt file from another driver and only change a few things?

Reply:
Yes, you can!  Grab any CMakeList.txt from the drivers directory.  The key portions
```
set(SOURCE_FILES
...Ai.xml and cpp files needed
)
add_fprime_module()
```
We'll avoid the UTs for now, as they are run automatically in CI, and that CI doesn't have an I2C.  So we'll take that discussion offline for now.